examples/bri-3/src/bri/state/state.profile.ts

```diff
@@ -0,0 +1,23 @@
+import { AutomapperProfile, InjectMapper } from '@automapper/nestjs';
+import { createMap, Mapper } from '@automapper/core';
+import { Injectable } from '@nestjs/common';
+import { StateTreeLeafValueContent } from './models/stateTreeLeafValueContent';
+import { StateTreeLeafValueContentDto } from './api/dtos/response/stateTreeLeafValueContent.dto';
+
+@Injectable()
+export class StateProfile extends AutomapperProfile {
+  constructor(@InjectMapper() mapper: Mapper) {
+    super(mapper);
+  }
+
+  override get profile() {
+    return (mapper: Mapper) => {
+      createMap(mapper, StateTreeLeafValueContent, StateTreeLeafValueContent);
```

Comment:
shouldn't one of them be `StateTreeLeafValueContentDto`?

Reply:
Nope, because of this issue https://github.com/eea-oasis/baseline/issues/740. Will be handled as part of mil5.

Reply:
@Therecanbeonlyone1969 @biscuitdey  please reapprove